backend/store/store.go

```diff
@@ -99,14 +99,18 @@ type SelectionPredicate struct {
 	// returned empty from the store, it indicates that there's no additional
 	// resources available
 	Continue string
+	// Offset indicates the index of the first resource to retrieve
+	Offset int64
 	// Limit indicates the number of resources to retrieve
 	Limit int64
 	// Subcollection represents a sub-collection of the primary collection
 	Subcollection string
+	// Ordering indicates the property to sort on, if supported by the store
+	Ordering string
```

Comment:
Can you also add `OrderingDirection string` (which will hold one of the constants you've defined fore `ASC/DESC`)?